packages/usa-validation/src/index.js

```diff
@@ -1,14 +1,44 @@
 const behavior = require("../../uswds-core/src/js/utils/behavior");
 const validate = require("../../uswds-core/src/js/utils/validate-input");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+
+const VALIDATE_INPUT = "input[data-validation-element]";
+const CHECKLIST_ITEM = `.${PREFIX}-checklist__item`;
 
 function change() {
   validate(this);
 }
 
-const validator = behavior({
-  "keyup change": {
-    "input[data-validation-element]": change,
+const validator = behavior(
+  {
+    "input change": {
+      [VALIDATE_INPUT]: change,
+    },
   },
-});
+  {
+    init(root) {
+      selectOrMatches(VALIDATE_INPUT, root).forEach((item) => {
```

Comment:
Naming each validation input as `item` can be easily confused with checklist items. Let's rename to make it clearer that each item is a validation input.

```suggestion
      selectOrMatches(VALIDATE_INPUT, root).forEach((input) => {
```

Reply:
Thanks. Updated in [34c5be9](https://github.com/uswds/uswds/pull/4914/commits/34c5be9f6793fce4cc4571e1631a2901c35d2c2d)